### ISSUE TITLE
Boot files are run from the volume you were looking at before shutdown.

### ISSUE THREAD

**Issue by TDW89** [KSP-KOS/KOS#790]
`switch to 2. reboot.` runs the boot file from volume 2.
This happens for shutting down from lack of power too.

Does not happen when reloading the craft from scene change.

**Comment by hvacengi** [comment]
I can't seem to replicate this bug.  Back in February I actually implemented the code that does what you're asking for, so I'd like to try and figure out what's going wrong on your version.  

I created a craft with two kOS units on it and launched.  I used a generic boot script that clears the screen.  Then I delete the boot file from 1, switch to 2, and reboot.  This leaves a boot.ks file on disk 2, but not on disk 1.  When the processor reboots, the boot script on 2 doesn't run for me (the boot up text remains visible), and `list.` shows my active volume as volume 1.

What version of kOS are you using?  (I'm using a developer version, which doesn't appear to be different, but I will go back to whatever version you're using to test).

Can you give me your steps to replicate the bug?  Was your code in the middle of another script, or is that already your most simplified code?  Do both units have a boot script?

If you press Ctrl-C to terminate the boot script, and then type `list.` does it identify you as being on volume 1 or 2?

Sorry for the barrage of questions, I'm just trying to get as much info as I can.

**Comment by TDW89** [comment]
Ok this is fixed in the dev build. I experienced it on the release build (0.16.2).
If you still want to repeat it for whatever reason here are the 2 boot scripts i just tested with.
boot_1.ks:
```
for core in ship:modulesnamed("kOSProcessor"). {
 core:doevent("open terminal").
}.
Print "I am core 1".
switch to 2.
Reboot.
```
boot_2.ks:
```
Print "I am core 2".
```

**Comment by hvacengi** [comment]
Thanks.  I'll test run it on on older versions to see whats up, just to satisfy my curiosity.  It looks like my fix should have been included in release 16.2, but maybe it didn't make it.  Glad we already had it fixed for you.
